backend/src/services/mqtt/mqtt.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable, OnApplicationBootstrap, Logger } from '@nestjs/common';
+import * as mqtt from 'mqtt';
+
+@Injectable()
+export class MqttService implements OnApplicationBootstrap {
+    private client: mqtt.MqttClient | undefined;
+    private readonly subscribed_topics = ['device_health/#', 'lock_status/#', 'alarm/#', 'lock/#'];
```

Comment:
do you think we should consolidate these in a JSON or object or enum so that its centralized? and so its easier to keep consistent with the embedded device side as well?